org.eclipse.scout.rt.client/src/main/java/org/eclipse/scout/rt/client/ui/basic/table/TableEventBuffer.java

```diff
@@ -544,9 +545,11 @@ protected static class TableEventMerger {
     private final Set<IColumn<?>> m_targetColumnSet;
     private final List<ITableRow> m_targetRows;
     private final Set<ITableRow> m_targetRowSet;
+    private final Map<ITableRow, Set<IColumn<?>>> m_targetUpdatedColumns;
 
     private List<IColumn<?>> m_mergedColumns;
     private List<ITableRow> m_mergedRows;
+    private boolean m_isUpdatedColumnsMerged = false;
```

Comment:
Necessary? The checks for m_mergedColumns/m_mergedRows in complete() already detect multiple invocations. The last "if" clause will never be evaluated.

If it should still remain, m_updatedColumnsMerged would be a better name (the "is" prefix is usually reserved for getters).

Reply:
True. I would also suggest to remove `m_mergedRows == null` as this will never be evaluated as well. Or am I missing something?